docs/conf.py

```diff
@@ -328,10 +328,15 @@ def build_schema_doc(key):
         s += '.. code-block:: yaml\n\n'
         temp = tempfile.NamedTemporaryFile()
         temp2 = tempfile.NamedTemporaryFile()
-        with open(temp.name, 'w') as ff:
-            json.dump(EXEMPLARS[key], ff, sort_keys=True)
-        jd = json.dumps(EXEMPLARS[key], sort_keys=True,
-                        indent=4, separators=(',', ': '))
+        documents = EXEMPLARS[key]
+        if not isinstance(documents, list):
```

Comment:
please use `collections.abc.Sequence` and test that it is not a string, instead of list

Reply:
Actually I'm making sure it isn't a single dict. I guess I could check if it is a mutable map.

Reply:
yes, that is much better